src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -369,10 +379,21 @@ export const securityAnalysisTableNmKContingenciesColumnsDefinition = (
             field: 'acceptableDuration',
             valueFormatter: (value: ValueFormatterParams) =>
                 convertDuration(value.data.acceptableDuration),
+            filterProps,
+            filterParams: {
+                filterDataType: FILTER_DATA_TYPES.NUMBER,
+                filterComparators: Object.values(FILTER_NUMBER_COMPARATORS),
+                parser: parseDuration,
```

Comment:
This parser function returns always '0' , so the filter is not working on this column

Reply:
Thanks, bug fixed in #2003